docset/windows/dism/Add-WindowsCapability.md

```diff
@@ -84,12 +84,6 @@ Accept wildcard characters: False
 
 ### -LogLevel
 Specifies the maximum output level shown in the logs.
```

Comment:
```suggestion
Specifies the maximum output level shown in the logs. The accepted values are as follows:
- Errors
- Warnings
- WarningsInfo (default)
```

Reply:
I removed it because it is actually included in the` Accepted Values: ` section.